### ISSUE TITLE
quiver in 3d?

### ISSUE THREAD

**Issue by imakn634** [Davide-sd/sympy-plot-backends#38]
I would like to know how to plot a (few) vector(s) in 3d.

We can plot a 2d quiver in SPB:
```python
from sympy import *
from spb import *

p=plot(xlim=(-2,2), ylim=(-2,2), aspect='equal', show=False)
ax=p.ax
ax.quiver([0],[0],[1],[1],
          angles='xy', scale_units='xy', scale=1);
```
![fig1](https://github.com/Davide-sd/sympy-plot-backends/assets/85376738/e91cb52a-43a5-46b3-aad9-2df1938b1e97)

It is essentially in the same way as in Matplotlib:
```python
import matplotlib.pyplot as plt
ax = plt.figure().add_subplot()
ax.set_aspect('equal')
ax.set_xlim(-2,2)
ax.set_ylim(-2,2)
ax.quiver([0],[0],[1],[1],
          angles='xy', scale_units='xy', scale=1);
```
![fig2](https://github.com/Davide-sd/sympy-plot-backends/assets/85376738/ca32e5c0-f8fe-4be4-92dd-19c49aaee530)

Matplotlib can also plot a 3d quiver:
```python
import matplotlib.pyplot as plt
ax = plt.figure().add_subplot(projection='3d')
ax.set_aspect('equal')
ax.set_xlim(-2,2)
ax.set_ylim(-2,2)
ax.set_zlim(-2,2)
ax.quiver([0],[0],[0], [1],[1],[1]);
```
![fig3](https://github.com/Davide-sd/sympy-plot-backends/assets/85376738/113c56d2-454b-47b4-b6c7-359afa8d7a87)

Therefore, I wonder if I can set `projection='3d'` option in SPB...  
Is it possible to plot a (few) vector(s) in 3d using SPB?

**Comment by Davide-sd** [comment]
For 2D arrows, you can use the `graphics` function together with `arrow_2d`. Take a look at [these examples](https://sympy-plot-backends.readthedocs.io/en/latest/modules/graphics/vectors.html#spb.graphics.vectors.arrow_2d).

Currently,  `arrow_3d` is not implemented. Maybe in the future, who knows.

**Comment by imakn634** [comment]
Thank you.  Our Jupyter server is still spb 2.4.3.   I will try the `graphics` function after upgrading to 3.0.1...

Long time before, you you answered me that once "ax" is set, we can set any detailed options (which are not implemented in spb) in the same way as in Matplotlib.  
`arrow_2d` is no implemented in spb 2.4.3, but we can plot 2d vectors using `ax.quiver()`.  
So I had a dream something like 
```python
from sympy import *
from spb import *

p=plot3d(xlim=(-2,2), ylim=(-2,2), show=False)
ax=p.ax
ax.quiver([0],[0],[0], [1],[1],[1]);
```
might work.  It does not work for now, but maybe in the future...

**Comment by Davide-sd** [comment]
I've just released a new version in which I have implemented the [`arrow_3d` function](https://sympy-plot-backends.readthedocs.io/en/latest/modules/graphics/vectors.html#spb.graphics.vectors.arrow_3d).

I'm closing this issue.
